### PR TITLE
Changed key filename

### PR DESCRIPTION
**Pull request by iulianpascalau**
- changed key filename
- updated scripts & README.md
- added default values in nodesSetup.json and genesis.json so the node can start with provided default config

Related to https://github.com/ElrondNetwork/elrond-deploy-go/pull/22

### CODE REVIEW

File: cmd/keygenerator/main.go [hunk 1]
```diff
@@ -56,15 +56,15 @@ VERSION:
...
+		Usage:       "What king of keys should generate. Available options: validator, wallet, both",
+		Value:       "validator",
 		Destination: &argsConfig.keyType,
 	}
 
 	argsConfig = &cfg{}
 
-	initialBalancesSkFileName = "initialBalancesSk.pem"
-	initialNodesSkFileName    = "initialNodesSk.pem"
+	initialBalancesSkFileName = "walletKey.pem"
```
Comment (SebastianMarian) [inline cmd/keygenerator/main.go:66]: walletKeyPemFileName ?

File: cmd/keygenerator/main.go [hunk 2]
```diff
@@ -56,15 +56,15 @@ VERSION:
...
+	initialNodesSkFileName    = "validatorKey.pem"
```
Comment (SebastianMarian) [inline cmd/keygenerator/main.go:67]: validatorKeyPemFileName ?

File: cmd/keygenerator/main.go (hunk 1 above)
Comment (iulianpascalau) [reply cmd/keygenerator/main.go:66]: yes, missed that :+1:

Comment (iulianpascalau) [reply cmd/keygenerator/main.go]: done

File: cmd/keygenerator/main.go (hunk 2 above)
Comment (iulianpascalau) [reply cmd/keygenerator/main.go]: done

**Pull request merged**